op-supervisor/supervisor/backend/depset/static.go

```diff
@@ -60,32 +62,93 @@ func NewStaticConfigDependencySetWithMessageExpiryOverride(dependencies map[eth.
 	return out, nil
 }
 
-// jsonStaticConfigDependencySet is a util for JSON encoding/decoding,
-// to encode/decode just the attributes that matter,
+// minStaticConfigDependencySet is a util for JSON/TOML encoding/decoding,
+// for just the minimal set of attributes that matter,
 // while wrapping the decoding functionality with additional hydration step.
-type jsonStaticConfigDependencySet struct {
-	Dependencies                map[eth.ChainID]*StaticConfigDependency `json:"dependencies"`
-	OverrideMessageExpiryWindow uint64                                  `json:"overrideMessageExpiryWindow,omitempty"`
+type minStaticConfigDependencySet struct {
+	Dependencies                map[string]*StaticConfigDependency `json:"dependencies" toml:"dependencies"`
+	OverrideMessageExpiryWindow uint64                             `json:"overrideMessageExpiryWindow,omitempty" toml:"override_message_expiry_window,omitempty"`
 }
```

Comment:
Renamed this struct so that its usable by json and toml methods. Also needed to change the map type to `map[string]*StaticConfigDependency` because the burntsushi/toml library requires map keys to be `string`

Reply:
This pr is making the assumption that the `minStaticConfigDependencySet` is what we will encode/decode to/from the chain config tomls. @Inphi can you confirm that contains all the data necessary for op-geth, op-node, and op-supervisor to load from the chain configs?

Reply:
The config looks good.